examples/minimum/variables.tf

```diff
@@ -1 +1,11 @@
-#(empty)
+variable "name" {
+  type        = string
+  description = "The name of the stack"
+  default     = "Minimum-SessionExample"
+}
+
+variable "create_session_preferences" {
+  type        = string
+  description = "Whether to create session preferences"
+  default     = true
```

Comment:
What session preferences will be created here? Seems to be creating an empty one...

Reply:
Added a feature